common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -3002,6 +3002,22 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     HIVE_TXN_READONLY_ENABLED("hive.txn.readonly.enabled", false,
       "Enables read-only transaction classification and related optimizations"),
 
+    HIVE_TXN_ACID_METRICS_MAX_CACHE_SIZE("hive.txn.acid.metrics.max.cache.size", 100,
```

Comment:
Could the descriptions be a bit longer and clearer? From users' perspective I don't think the meanings of the configs are obvious.